fixbackend/workspaces/router.py

```diff
@@ -58,6 +60,7 @@ async def get_workspace(
         workspace_id: WorkspaceId,
         user: AuthenticatedUser,
         workspace_repository: WorkspaceRepositoryDependency,
+        _: Annotated[bool, Depends(WorkspacePermissionChecker(WorkspacePermission.read))],
```

Comment:
I like the flexibility of this approach. Can you think of a way to shorten the syntax?